### ISSUE TITLE
how to check out my model?

### ISSUE THREAD

**Issue by Zweiyuan** [fizyr/keras-retinanet#916]
hi, I have some questions about test. I used csv data.
I had finished training and I got a '.h5' file. But I don't know how to test my model.

Or how to evaluate my model?
Such as 'accuracy on test data' or **'precision'**, **'recall'**,  **'F1-score'**,  **'support'**. 

I only know the loss from:
```
Epoch 50/50
10000/10000 [==============================] - 8702s 870ms/step - loss: 0.0204 - regression_loss: 0.0204 - classification_loss: 2.6087e-05

Epoch 00050: saving model to ./snapshots/resnet50_csv_50.h5
```

**Comment by hgaiser** [comment]
There is an evaluate.py tool for computing mAP scores. The rest of the scores you mentioned you will have to compute yourself.

**Comment by SalahAdDin** [comment]
@Zweiyuan , did you do it? I need also evaluate this scores, and the GPU consumption.

**Comment by Zweiyuan** [comment]
@SalahAdDin I got the scores by `keras_retinanet\utils\eval.py`

you can modify the function "evaluate" in "eval.py", make it output what you need.

**Comment by SalahAdDin** [comment]
@Zweiyuan can you provide a pull request? i did some changes in mine but i'm not sure they are correct.
Also, i cannot evaluate the model because of [this bug](https://github.com/fizyr/keras-retinanet/issues/1152).

**Comment by Zweiyuan** [comment]
@SalahAdDin Sorry i’m busy these days. You should add `--convert-model` after your cmd command.
